docker-for-aws/faqs.md

```diff
@@ -150,3 +150,10 @@ $ sudo ping 10.0.0.4
 ```
 
 > **Note**: Access to Docker for AWS and Azure happens through a shell container that itself runs on Docker.
+
+## How do I uninstall Docker for AWS?
+
+You can remove the Docker for AWS setup and stacks through the [AWS
+Console](https://console.aws.amazon.com/console/home){: target="_blank"
+class="_"} on the CloudFormation page. See [Uninstalling or removing a
```

Comment:
I think this should not have the gerund form either here or at the actual anchor location.

Reply:
I'm going to leave headers as written